dcos_test_utils/dcos_cli.py

```diff
@@ -92,6 +94,45 @@ def clear_cli_dir():
         if os.path.exists(path):
             shutil.rmtree(path)
 
+    def exec(self, cmd: str, stdin=None, check=True) -> subprocess.CompletedProcess:
```

Comment:
Why have `check`?

Reply:
While testing I would like to be able to run commands that I know will fail, and assert on their exit, without having to `rescue` from the exception. There is a behavior on the `subprocess` library that is to pass a boolean called `check` that when True, does not check for the command terminating with status code 0. I want to be able to configure this from the client of the exec function.